### PR TITLE
Vaccination Info Card Update (EXPOSUREAPP-9203)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Updated Vaccination Info Card

Info Card: **All Vaccinations are done and waiting period is also over**
image

Info Card: **Booster Eligible**
image

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt [hunk 1]
```diff
@@ -22,21 +26,62 @@ class VaccinationInfoCard(parent: ViewGroup) :
...
+                    R.string.vaccination_card_booster_eligible_faq,
+                    R.string.vaccination_card_booster_eligible_faq_link_container,
+                    R.string.vaccination_card_booster_eligible_faq_link
+                )
+            }
+
+            VaccinatedPerson.Status.COMPLETE,
+            VaccinatedPerson.Status.IMMUNITY -> {
+                body.text = context.resources.getString(
+                    R.string.vaccination_list_immunity_card_body
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:72]: The message is not the same for COMPLETE and IMMUNITY states , right ?

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:72]: yes, need to check what's the difference. Will be adapted :)

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:71]: is done

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt [hunk 2]
```diff
@@ -22,21 +26,63 @@ class VaccinationInfoCard(parent: ViewGroup) :
...
+            VaccinatedPerson.Status.BOOSTER_ELIGIBLE -> {
+
+                body.text = context.resources.getString(
+                    R.string.vaccination_card_booster_eligible,
+                    boosterRule!!.rule.identifier
+                )
+
+                body2Faq.isVisible = true
+                boosterBadge.isVisible = true
+                body2Faq.setTextWithUrl(
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:62]: it should be visible only when `vaccinatedPerson.hasBoosterNotification` is true,
i.e `boosterBadge.isVisible=vaccinatedPerson.hasBoosterNotification`

**Review by mtwalli — COMMENTED** [review]
Please update screenshots also ?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt (hunk 2 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:62]: good point! Will be added :)

**Review by jurajkusnier — COMMENTED** [review]
LGTM, besides the visibility thing that Mohamed mentioned

**Comment by SamuraiKek** [comment]
> LGTM, besides the visibility thing that Mohamed mentioned

Same from my side.

**Comment by axelherbstreith** [comment]
@mtwalli thank you for fixing it 🥇

**Pull request merged**